src/components/ListItem.jsx

```diff
@@ -1,5 +1,45 @@
+import { useEffect, useState } from 'react';
+import { updateItem } from '../api/firebase';
+import { ONE_DAY_IN_MILLISECONDS } from '../utils';
 import './ListItem.css';
 
-export function ListItem({ name }) {
-	return <li className="ListItem">{name}</li>;
+export function ListItem({ listPath, item, name }) {
+	const [isChecked, setIsChecked] = useState(false);
+
+	function handleChange(e) {
```

Comment:
nit: "e" seems not to be used anywhere within the function. I would suggest removing it.

Reply:
Good spot thanks Iryna! We will remove this parameter.